docs/en_US/Tutorial/ExperimentConfig.md

```diff
@@ -592,6 +592,14 @@ Specifies the pre-command that will be executed before the remote machine execut
 
 __Note__: Because __preCommand__ will execute before other commands each time, it is strongly not recommended to set __preCommand__ that will make changes to system, i.e. `mkdir` or `touch`.
 
+### remoteConfig
```

Comment:
it is a little strange to have both "machineList" and "remoteConfig" in the same level

Reply:
machineList is a list type field, I considered merge machineList under remoteConfig, but it may cause compatibility problem.